openmetadata-ui/src/main/resources/ui/src/utils/CustomizableLandingPageUtils.tsx

```diff
@@ -55,13 +55,17 @@ export const getAddWidgetHandler =
     if (
       placeholderWidgetKey === LandingPageWidgetKeys.EMPTY_WIDGET_PLACEHOLDER
     ) {
+      const emptyWidgetPlaceholder = currentLayout.find(
+        (item) => item.i === LandingPageWidgetKeys.EMPTY_WIDGET_PLACEHOLDER
+      ) ?? { x: 0, y: 99 };
+
       return [
         ...moveEmptyWidgetToTheEnd(currentLayout),
         {
           w: widgetWidth,
           h: widgetHeight,
-          x: 0,
-          y: 0,
+          x: emptyWidgetPlaceholder.x,
+          y: emptyWidgetPlaceholder.y,
```

Comment:
We are giving the x and y the same as the empty placeholder, shouldn't the 'y' be y-1?

Reply:
It will be fine as the new widget will override the empty placeholder one!